### PR TITLE
Fix Documentation sidebar links

### PR DESCRIPTION
**Pull request by snapecode**
Adjusted the sidebar Link Levels and added a missing modify colors link.

### CODE REVIEW

**Comment by shahednasser** [comment]
Thank you for your work. It seems there's an additional problem in the `SidebarLink component` here:

https://github.com/sButtons/sbuttons/blob/ab79052fd3c0b7e9fcbbaa02fa55ef0843f4f3a0/website/src/components/SidebarLink.js#L31-L35

can you please change `ml-${level * 3}` to `ml-${level ? level + 3 : 0}` ? I think it would be more fitting.

Also, please remove all changes in package-lock.json

**Comment by snapecode** [comment]
Hi @shahednasser, I made the change to sidebar Link and reverted package-lock. Is the pull request correct? I feel like I may have merged an old branch. Please keep recommending little changes so I can practice these push and pulls.
The sidebar does still seem a little out of place and not perfectly aligned, and the footer pushing in from the bottom. What do you think?

**Review by shahednasser — CHANGES_REQUESTED** [review]
It seems that you reverted the changes you did as well with the sidebarlink levels, that's why it still seems not aligned correctly. Please add the changes you added before to the sidebarlink levels

File: website/src/pages/Documentation/index.js [hunk 1]
```diff
@@ -21,20 +21,21 @@ function Documentation () {
...
+                    <SidebarLink to="#less-modify-colors" text="Modify Colors" level={1} />
                     <SidebarLink to="#usage" text="Usage" />
                     <SidebarLink to="#basic-button" text="Basic Button" level={1} />
                     <SidebarLink to="#block-buttons" text="Block Buttons" level={1} />
                     <SidebarLink to="#disabled-buttons" text="Disabled Buttons" level={1} />
                     <SidebarLink to="#icon-buttons" text="Icon Buttons" level={1} />
-                    <SidebarLink to="#only-icon" text="Only Icon" level={2} />
-                    <SidebarLink to="#icon-left" text="Icon Left" level={2} />
-                    <SidebarLink to="#icon-right" text="Icon Right" level={2} />
+                    <SidebarLink to="#only-icon" text="Only Icon" level={4} />
```
Comment (shahednasser) [inline website/src/pages/Documentation/index.js:30]: Only icon, icon-left and icon-right should be level 2

File: website/src/pages/Documentation/index.js [hunk 2]
```diff
@@ -21,20 +21,21 @@ function Documentation () {
...
+                    <SidebarLink to="#icon-left" text="Icon Left" level={4} />
+                    <SidebarLink to="#icon-right" text="Icon Right" level={4} />
                     <SidebarLink to="#toggle-buttons" text="Toggle Button" level={1} />
-                    <SidebarLink to="#toggle-button-checkbox" text="Toggle Button as Checkboxes" level={2} />
-                    <SidebarLink to="#toggle-button-radio" text="Toggle Button as Radio Buttons" level={2} />
+                    <SidebarLink to="#toggle-button-checkbox" text="Toggle Button as Checkboxes" level={4} />
+                    <SidebarLink to="#toggle-button-radio" text="Toggle Button as Radio Buttons" level={4} />
```
Comment (shahednasser) [inline website/src/pages/Documentation/index.js:35]: toggle-button-checkbox and toggle-button-radio should be level 2

File: website/src/pages/Documentation/index.js [hunk 3]
```diff
@@ -21,20 +21,21 @@ function Documentation () {
...
                     <SidebarLink to="#dark-mode" text="Dark Mode" level={1} />
-                    <SidebarLink to="#dark-mode-class" text="Using dark-mode Class" level={2} />
-                    <SidebarLink to="#dark-mode-attribute" text="Using data-theme attribute" level={2} />
+                    <SidebarLink to="#dark-mode-class" text="Using dark-mode Class" level={4} />
```
Comment (shahednasser) [inline website/src/pages/Documentation/index.js:37]: dark-mode-class and dark-mode-attribute should be level 2

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: website/src/components/SidebarLink.js [hunk 4]
```diff
@@ -31,7 +31,7 @@ function SidebarLink ({text, to = "", level = 0, clickCallback = null, current =
     return (
         <Link to={to.length ? to : '#'} 
             onClick={scroll}
-            className={`block ${active ? 'border-red-600' : ''} sidebar-item lg:w-max text-lg border-b-2 border-transparent hover:border-red-600 dark:text-white ml-${level * 3}`}>{text}</Link>
+            className={`block ${active ? 'border-red-600' : ''} sidebar-item lg:w-max text-lg border-b-2 border-transparent hover:border-red-600 dark:text-white ml-${level ? level + 3 : 0}`}>{text}</Link>
```
Comment (shahednasser) [inline website/src/components/SidebarLink.js:34]: sorry for the delay @snapecode but can you please change this to `level + 2`?

**Comment by shahednasser** [comment]
Great work thank you!

@all-contributors please add @snapecode for code and bug

**Pull request merged**